about.markdown

```diff
@@ -1,18 +1,73 @@
 
-[jekyll-organization]: https://github.com/jekyll
+## Why did we change our name from Rails Girls Berlin to code curious?
+
+There were a number of reasons, all described in full in [this article](../_posts/2017-10-25-rails-girls-berlin-is-growing-up.md). Briefly, the main reasons for the change were that we wanted to:
+- be open towards other technologies and programming languages,
+- address women, trans and non-binary people instead of girls,
+- highlight a playful aspect of coding.
+
+## Why are so few women in IT?
+
+Due to a **lack of female role models**, women are often prejudiced towards Computer Science as a career as it is commonly **perceived as male dominated, isolated, uncreative and uncommunicative**. In reality, programming demands a lot of creativity as well as good communication skills in order to build a good product.
+
+Furthermore, it is a common **misbelief that being good at Maths is a prerequisite** for learning programming. Due to outdated undergraduate teaching methods that favour competition over cooperation, girls are often less interested in Mathematics. They therefore often show a lower performance at school which keeps them from taking up an IT career later in life.
+
+
+## How we are trying to solve this problem
+
+**We make coding social** by creating a communicative and cooperative atmosphere around it. Our workshops kick off with an installation get-together and are provided with food that attendees bring along to share.
+
+By teaching women in groups of two, we create an **environment without pressure or competition** which diminishes the fear of asking “stupid” questions.
+
+**We connect women with each other, with mentors / coaches and role models** who have shown that it doesn’t need an academic background in computer science or be good at Maths to become a successful developer.
+
+**We foster continued learning** by encouraging attendees to join project groups after the workshop. These groups are led by experienced developers and stretch over several weeks to months. In addition to that, learners of all levels come together regularly on our “Code & Cake”.
+
+
+## What do we mean by „women“?
+
+We welcome people who *identify as women* of *any ethnicity and background above the age of 16*. No matter if you’re still in high school, have been working for many years or are staying home to take care of relatives. Diversity matters to us, so if you are not fluent in English, don’t own a laptop, can’t find someone to care of your children during the workshop, please contact us and we will try to find a solution.
+
+## Who we are
+
+Code curious Berlin events are organized by a small group of organisers and a huge group of programmers. Everyone is volunteering for this project, because we believe that women are the future of programming. The overwhelmingly positive feedback we are receiving from our attendees and coaches is convincing us to keep this project going.
```

Comment:
Is it code curious, Code curious or code curious? (also in other occasions)

Reply:
Or is it Code curious because it's the beginning of a sentence? :D

Reply:
At the beginning of the whole name change process we all agreed on the writing "code curious". But I feel like already with the logo we didn't stay consistent anymore (CODE CURIOUS). Maybe we need to make a little wiki on our own conventions for these things...? Also especially in German we should agree on some kind of gender neutral forms for all kind of people-words.

Reply:
If our name is `code curious` then we should consistently use it as `code curious` For example `iTunes` and `eBay` Don't get capitalised at beginning of sentence

Reply:
I think that's the best way, but the logo is the logo and we can't change that anymore without people questioning our sanity I think.

Reply:
I'm fine with our logo, and having a separate convention for written form.